test/com/ableton/VirtualEnvTest.groovy

```diff
@@ -40,7 +40,8 @@ class VirtualEnvTest extends BasePipelineTest {
   @Test
   void newObjectWithNullScript() throws Exception {
     try {
-      def venv = new VirtualEnv(null, 'python2.7')
+      @SuppressWarnings('UnusedVariable')
+      def unused = new VirtualEnv(null, 'python2.7')
```

Comment:
Why not `new VirtualEnv(null, 'python2.7')` only?

Reply:
Hmm, good point. I had avoided doing that because my IDE would throw a warning with `Result of VirtualEnv() is ignored`, but it seems that codenarc doesn't care about this so I guess it is fine. 😅 

(Same goes for the same comment in other lines)